packages/components/src/components/breadcrumb/component.tsx

```diff
@@ -24,14 +24,14 @@ export class KolBreadcrumb implements KoliBriBreadcrumbAPI {
 				{index === lastIndex ? (
 					<span>
 						{hideLabel ? (
-							<kol-icon _ariaLabel={link._label} _icon={typeof link._icon === 'string' ? link._icon : 'codicon codicon-symbol-event'} />
+							<kol-icon _ariaLabel={link._label || link._href} _icon={typeof link._icon === 'string' ? link._icon : 'codicon codicon-symbol-event'} />
```

Comment:
keine änderungen in diese Komponente nötig

Reply:
TS sagt: "Type 'string | undefined' is not assignable to type 'string'." 
_ariaLabel ist required, link._label jedoch nicht.

Reply:
in der Bradcrump ist das label an den links pflicht. Daher können die LinkProps nicht mehr verwendet werden. Hier wird ein neuer Typ nötig, der zwar viel vom Link nutzt, aber nicht optioanale die label prop hat.

Könnte dann so aussehen:
```ts
type BreadcrumLinkProps  = Generic.Element.Members<RequiredLinkProps & { label: string }, Omit<OptionalLinkProps, "label">>;
```